server/app/auth/oidc/OidcCiviFormProfileAdapter.java

```diff
@@ -97,7 +107,7 @@ public CiviFormProfileData mergeCiviFormProfile(
   }
 
   /** Create a totally new CiviForm profile from the provided OidcProfile. */
```

Comment:
Need to update this doc as the new profile isn't based on the OidcProfile anymore.  This kinda "smells" though which likely means we need more refactoring after this PR.

Reply:
The new profile is potentially dependent on the OidcProfile, because the AdfsProfileAdapter uses it to decide whether to create a global admin or program admin.

Reply:
Let's leave it as is in this PR, but it's an oddity that usually is a hint/smell that the abstractions aren't correct.

Update the comment though to break that the new profile is "from" the Oidc one since that's not necessarily the case anymore.  Maybe "informed by the OidcProfile"